### PR TITLE
release-0.10.2

### PR DESCRIPTION
**Pull request by dkent600**

### CODE REVIEW

**Comment by EzraWeller** [comment]
@tibetsprague 
On rinkeby, when I click on the address of an unsupported scheme, I am getting this error:
Annotation 2020-03-16 182537
Annotation 2020-03-16 182536
Is this an issue with #1501 or am I misinterpreting what that was meant to change?

**Comment by EzraWeller** [comment]
Getting an error on the redemption page on rinkeby:
Redeem page error
(clicked on "view all" from the redemption drop down)

**Comment by dkent600** [comment]
@EzraWeller Were you going to create tickets for bugs you find?

**Comment by EzraWeller** [comment]
I wasn't sure if it would be worth in these cases -- it would be important for me to reference any new issues I create in this comment section, right? To keep track of the issues related to this release? Or should I track them another way?

**Comment by dkent600** [comment]
> I wasn't sure if it would be worth in these cases -- it would be important for me to reference any new issues I create in this comment section, right? To keep track of the issues related to this release? Or should I track them another way?

I'm not sure, but I'm leaning toward referencing each ticket in the description of this PR.  Then is appropriate, we could prepend each ticket reference with "Resolves:" so github will automatically close the ticket when this PR is merged.

**Comment by EzraWeller** [comment]
Okay, I'm ready approve this. #1519 is still open, but it doesn't seem to be an issue on mainnet (b/c they aren't as many pending redemptions?).

**Comment by tibetsprague** [comment]
@EzraWeller yeah I’m thinking about that one but not an issue on prod yet

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
   - Features Added
   - Bugs Fixed
 
+## 0.10.2
+  - Features Added
+    - Fix bug when entering numbers for uint256 data types in the GenericScheme new proposal modal 
+
+  - Bugs Fixed
+
```
Comment (orenyodfat) [inline CHANGELOG.md]: how about the animation pr ?

Comment (dkent600) [reply CHANGELOG.md]: https://github.com/daostack/alchemy/pull/1526

**Comment by dkent600** [comment]
> this increase Heroku slug size with around 180mega.
> how come ?

@orenyodfat  See: https://github.com/daostack/alchemy/pull/1528

**Pull request merged**